wren-ui/src/apollo/server/repositories/relationshipRepository.ts

```diff
@@ -187,25 +186,4 @@ export class RelationRepository
     );
     return result.map((r) => this.transformFromDBData(r)) as RelationInfo[];
   }
-
-  public async findExistedRelationBetweenModels(modelIds) {
```

Comment:
I think this should be rewrite instead of deleted.
Can rewrite to check if columnIds exist in the table.

Reply:
Hmm... I rewrote the checking logic to check if it is a duplicate relationship